frontend/src/components/bucket/BucketConfigForm.vue

```diff
@@ -77,14 +77,31 @@ const onSubmit = async (values: any) => {
       formBucket.key = joinPath(values.key);
     }
 
+    const bucketChanges = differential(formBucket, initialValues);
+
     props.bucket
-      ? await bucketStore.updateBucket(props.bucket?.bucketId, differential(formBucket, initialValues))
+      ? await bucketStore.updateBucket(props.bucket?.bucketId, bucketChanges)
       : await bucketStore.createBucket(formBucket);
 
     await bucketStore.fetchBuckets({ userId: getUserId.value, objectPerms: true });
+
+    // trim trailing "//", if present
+    // (modifying getBucketPath() instead seems to break nested bucket display [PR-146])
```

Comment:
i see the issue here. we could modify the getBucketPath function to remove the trailing slash.. but that will impact the other uses of that function.